src/main/java/org/generationcp/middleware/dao/MethodDAO.java

```diff
@@ -606,4 +614,23 @@ private Criteria setCriteriaFilteredMethods(final BreedingMethodSearchRequest me
 		}
 		return criteria;
 	}
+
+	public long countByVariable(final int variableId){
+		try {
+			final SQLQuery query = this.getSession().createSQLQuery(MethodDAO.COUNT_BREEDING_METHODS_WITH_VARIABLE);
+			query.setParameter("variableId", variableId);
```

Comment:
Is it necessary to set the `variableId` 6 times since we are setting value by same parameter name?

Reply:
done.